packages/odyssey-react-mui/src/Card.tsx

```diff
@@ -170,7 +153,22 @@ const Tile = ({
   );
 };
 
-const MemoizedTile = memo(Tile);
-MemoizedTile.displayName = "Tile";
+const MemoizedCard = memo(Card);
+MemoizedCard.displayName = "Card";
+
+/**
+ * @deprecated The 'Tile' component is now called 'Card'. Please update your references as 'Tile' will be deprecated soon.
+ */
+const Tile = (props: CardProps) => {
+  useEffect(() => {
+    console.warn(
+      "Warning: The 'Tile' component is now called 'Card'. Please update your references as 'Tile' will be deprecated soon.",
+    );
+  }, []);
+
+  return <MemoizedCard {...props} />;
+};
+
+Tile.displayName = "Tile";
```

Comment:
❤️ This is great! Thanks for adding in this deprecation.

As cumbersome as it is, we should _also_ memoize `Tile`. Enduser will be using that component, so we'll want to make sure we don't impact their performance.

The compiler can't come sooner!